resources/META-INF/optional-groovy-dependencies.xml

```diff
@@ -32,12 +32,14 @@
     </extensions>
 
     <actions>
-        <action id="hybris.groovy.execute" class="com.intellij.idea.plugin.hybris.groovy.actions.GroovyExecuteAction"/>
+        <action id="hybris.groovy.execute.commit.mode.off" class="com.intellij.idea.plugin.hybris.groovy.actions.CommitModeOffGroovyExecuteAction"/>
+        <action id="hybris.groovy.execute.commit.mode.on" class="com.intellij.idea.plugin.hybris.groovy.actions.CommitModeOnGroovyExecuteAction"/>
```

Comment:
`GroovyExecuteInCommitModeOnAction`

let's have Language prefix for file, it will be easier to find and group